docs/concepts/README.md

```diff
@@ -1,4 +1,139 @@
 # Concepts
 
-This section of the documentation helps you learn about the JobSet API and its semantics.
+A JobSet creates one or more Jobs. It allows you to create sets of jobs of different or identical templates, and control their lifecycle together. 
+
+## Running an Example JobSet
+
+Here is an example JobSet. It runs a distributed PyTorch training workload.
+
+```yaml
+apiVersion: jobset.x-k8s.io/v1alpha1
+kind: JobSet
+metadata:
+  name: pytorch
+spec:
+  replicatedJobs:
```

Comment:
This is still `jobs` in the JobSetSpec API definition, we need to change it to `replicatedJobs`

Reply:
I was referring to the json/yaml representation, it still says `json:"jobs"`, it needs to change to `json:"replicatedJobs"` for this example to work.

Reply:
that is a bug we should fix :)

sending a PR in a sec